.travis.yml

```diff
@@ -17,9 +17,8 @@ script:
   - docker run test tests/test_integration.py
   - >
     docker run
-    -e TRELLO_API_KEY=$TRELLO_API_KEY
-    -e TRELLO_TOKEN=$TRELLO_TOKEN
-    -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID
+    -e MONGO_DB_USER_NAME=$MONGO_DB_USER_NAME
```

Comment:
It would be good to remove the Trello secrets set in Travis now that we're not using them